python/lsst/ctrl/bps/clustered_quantum_graph.py

```diff
@@ -260,11 +260,10 @@ def get_cluster(self, name):
             Raised if the ClusteredQuantumGraph does not contain
             a cluster with given name.
         """
-        if name not in self._cluster_graph:
-            raise KeyError(f"{self.name} does not have a cluster named {name}")
-
-        _LOG.debug("get_cluster nodes = %s", list(self._cluster_graph.nodes))
-        attr = self._cluster_graph.nodes[name]
+        try:
+            attr = self._cluster_graph.nodes[name]
+        except KeyError as ex:
+            raise KeyError(f"{self.name} does not have a cluster named {name}") from ex
```

Comment:
Shouldn't it read `self._name` (note the leading underscore)?

Reply:
There's a property self.name?

Reply:
Never mind then. I guess I got confused by the way GitHub displays the diff for this file.